cylc/flow/cfgspec/workflow.py

```diff
@@ -1730,57 +1730,37 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                 ''')
 
             with Conf('workflow state polling', desc=f'''
-                Configure automatic workflow polling tasks as described in
-                :ref:`WorkflowStatePolling`.
+                Deprecated support for automatic workflow state polling tasks
+                as described in :ref:`WorkflowStatePolling`. Note the Cylc 7
+                "user" and "host" config items are not supported.
 
-                The items in this section reflect
-                options and defaults of the ``cylc workflow-state`` command,
-                except that the target workflow ID and the
-                ``--task``, ``--cycle``, and ``--status`` options are
-                taken from the graph notation.
-
-                .. versionchanged:: 8.0.0
+                .. versionchanged:: 8.3.0
 
                    {REPLACES}``[runtime][<namespace>]suite state polling``.
-            '''):
-                Conf('user', VDR.V_STRING, desc='''
-                    Username of your account on the workflow host.
 
-                    The polling
-                    ``cylc workflow-state`` command will be
-                    run on the remote account.
-                ''')
-                Conf('host', VDR.V_STRING, desc='''
-                    The hostname of the target workflow.
+                .. deprecated:: 8.3.0
 
-                    The polling
-                    ``cylc workflow-state`` command will be run there.
-                ''')
+                   Please switch to workflow_state xtriggers.
+            '''):
                 Conf('interval', VDR.V_INTERVAL, desc='''
                     Polling interval.
                 ''')
                 Conf('max-polls', VDR.V_INTEGER, desc='''
-                    The maximum number of polls before timing out and entering
-                    the "failed" state.
+                    Maximum number of polls to attempt before the task fails.
+                ''')
+                Conf('output', VDR.V_STRING, desc='''
+                    Target task output (trigger name, not task message).
+                    Alternatively, put this in the graph trigger syntax.
                 ''')
```

Comment:
I think I was trying to fix this config so that it matches the (old-format CLI args), which it was always supposed to do. But yeah, no one could be relying on this item in existing usage, and we're deprecating the feature, so adding it is arguably a bad idea... I'll remove it.